multipatch_analysis/pipeline/multipatch/dataset.py

```diff
@@ -77,6 +77,7 @@ def create_db_entries(cls, job, session):
                     baseline_current=rec.baseline_current,
                     baseline_rms_noise=rec.baseline_rms_noise,
                     qc_pass=qc_pass,
+                    meta={'qc_failures': qc_failures},
```

Comment:
To save on space, make `meta=None` if there were no qc failures. (I'm worried with millions of PRs, this can significantly impact the DB size)

Reply:
I assume this should be every place that meta is being set with qc_failures

Reply:
correct.